pkg/cmd/cli/install/install.go

```diff
@@ -85,7 +84,6 @@ type Options struct {
 	DefaultSnapshotMoveData         bool
 	DisableInformerCache            bool
 	ScheduleSkipImmediately         bool
-	MaintenanceCfg                  repository.MaintenanceConfig
```

Comment:
Please also remove this same var name from `type VeleroOptions struct {`

in pkg/install/resources.go

This should make this PR also resolve https://github.com/vmware-tanzu/velero/issues/8157

Reply:
this var appears to be unused and not referenced anywhere there.